code/modules/mob/living/carbon/xenomorph/castes/Boiler.dm

```diff
@@ -9,7 +9,7 @@
 	plasma_gain = XENO_PLASMA_GAIN_TIER_7
 	plasma_max = XENO_PLASMA_TIER_4
 	xeno_explosion_resistance = XENO_EXPLOSIVE_ARMOR_TIER_2
-	armor_deflection = XENO_ARMOR_TIER_2
+	armor_deflection = XENO_ARMOR_TIER_1
```

Comment:
If you are changing armour of baseline caste, change the modifier for the trapper accordingly, or this happens.
![image](https://user-images.githubusercontent.com/4447185/231749243-3da7602a-e267-4b9e-8e17-c25f39356f67.png)